lib/src/main/kotlin/com/outdatedversion/survival/command/MessageCommand.kt

```diff
@@ -0,0 +1,72 @@
+package com.outdatedversion.survival.command
+
+import co.aikar.commands.BaseCommand
+import co.aikar.commands.annotation.*
+import com.outdatedversion.survival.MessagingModule
+import com.outdatedversion.survival.Plugin
+import com.outdatedversion.survival.asText
+import com.outdatedversion.survival.model.PointOfInterest
+import com.outdatedversion.survival.model.asComponent
+import net.kyori.adventure.text.Component
+import net.kyori.adventure.text.format.NamedTextColor
+import net.kyori.adventure.text.format.TextDecoration
+import org.bukkit.Bukkit
+import org.bukkit.NamespacedKey
+import org.bukkit.entity.Player
+import org.bukkit.persistence.PersistentDataType
+import java.util.*
+
+@CommandAlias("msg|tell|whisper")
```

Comment:
what do you think about adding `m` as well?

Reply:
Added m